src/Templates/NServiceBusWindowsService/.template.config/template.json

```diff
@@ -19,6 +19,14 @@
       "description": "The target framework for the project.",
       "datatype": "choice",
       "choices": [
+        {
+          "choice": "netcoreapp2.1",
+          "description": "Target netcoreapp2.1"
```

Comment:
is there any good reason to provide a `netcoreapp2.0` choice?

Reply:
I don't think so, since .NET Core 2.0 is completely dead and no longer supported.

All of the other frameworks listed here are still supported by Microsoft.

Reply:
Fine then.